configs/components/_base-ruby-augeas.rb

```diff
@@ -20,6 +20,11 @@
 pkg.build_requires "augeas"
 
 pkg.environment "PATH", "$(PATH):/opt/pl-build-tools/bin:/usr/local/bin:/opt/csw/bin:/usr/ccs/bin:/usr/sfw/bin"
+
+if platform.architecture == 'arm64' && platform.is_macos? && platform.os_version.to_i >= 13
+  pkg.environment "PATH", "/opt/homebrew/bin:$(PATH):/opt/pl-build-tools/bin:/usr/local/bin:/opt/csw/bin:/usr/ccs/bin:/usr/sfw/bin"
```

Comment:
I think that here you don't need to repeat `/opt/pl-build-tools/bin:/usr/local/bin:/opt/csw/bin:/usr/ccs/bin:/usr/sfw/bin`. You can just do `"/opt/homebrew/bin:$(PATH)"`, similar to how [this line](https://github.com/puppetlabs/puppet-runtime/blob/master/configs/components/_base-ruby-augeas.rb#L29) is written.

Reply:
We can exclude this change after rebasing, see https://github.com/puppetlabs/puppet-runtime/pull/680/files#diff-e9d1d8770ae2de7f9bd9522753b5be0e02ce8b3020b57266e7f5d89ff4766f19